.github/workflows/reusable-build.yml

```diff
@@ -279,7 +282,7 @@ jobs:
         run: exit 0
 
   build_iso:
```

Comment:
`build_iso` also contains the matrix, so I think we should fix that before this PR is merged.
Otherwise we will build ALL images 5 times over every time a PR is merged if I am understanding this correct?

Reply:
Ok, reviewed the yaml there...

While I agree with you it needs to be fixed, the build_iso job only runs on testing branch, and it's already a problem as it doesn't support aurora... 

Per Discord chat with @m2Giles , we are thinking merge this now, and he will tackle the Build ISO workflow improvements.

Reply:
Great catch.